src/main/java/org/isf/accounting/gui/PatientBillEdit.java

```diff
@@ -805,148 +923,120 @@ private JScrollPane getJScrollPaneTotal() {
-			jTableTotal.getColumnModel().getColumn(1).setMaxWidth(CURRENCY_CODE_WIDTH);
-			jTableTotal.getColumnModel().getColumn(2).setMinWidth(PRICE_WIDTH);
-			jTableTotal.getColumnModel().getColumn(2).setMaxWidth(PRICE_WIDTH);
-			jTableTotal.setMaximumSize(new Dimension(PANEL_WIDTH, TOTAL_HEIGHT));
-			jTableTotal.setMinimumSize(new Dimension(PANEL_WIDTH, TOTAL_HEIGHT));
-			jTableTotal.setPreferredSize(new Dimension(PANEL_WIDTH, TOTAL_HEIGHT));
+			jTableTotal = new JTable(new JTableTotalModel());
+			sizeJTableTotal();
 		}
 		return jTableTotal;
 	}
 
+	private void setJTableTotal() {
+		jTableTotal.setModel(new JTableTotalModel());
+		sizeJTableTotal();
+	}
+
+	private void sizeJTableTotal() {
+		jTableTotal.getColumnModel().getColumn(1).setMinWidth(CURRENCY_CODE_WIDTH);
+		jTableTotal.getColumnModel().getColumn(1).setMaxWidth(CURRENCY_CODE_WIDTH);
+		jTableTotal.getColumnModel().getColumn(2).setMinWidth(PRICE_WIDTH);
+		jTableTotal.getColumnModel().getColumn(2).setMaxWidth(PRICE_WIDTH);
+		jTableTotal.setMaximumSize(TOTAL_TABLE_SIZE);
+		jTableTotal.setMinimumSize(TOTAL_TABLE_SIZE);
+		jTableTotal.setPreferredSize(TOTAL_TABLE_SIZE);
+	}
+
 	private JScrollPane getJScrollPanePayment() {
 		if (jScrollPanePayment == null) {
 			jScrollPanePayment = new JScrollPane();
 			jScrollPanePayment.setBorder(null);
 			jScrollPanePayment.setViewportView(getJTablePayment());
-			jScrollPanePayment.setMaximumSize(new Dimension(PANEL_WIDTH, PAYMENT_HEIGHT));
-			jScrollPanePayment.setMinimumSize(new Dimension(PANEL_WIDTH, PAYMENT_HEIGHT));
-			jScrollPanePayment.setPreferredSize(new Dimension(PANEL_WIDTH, PAYMENT_HEIGHT));
+			Dimension size = new Dimension(PANEL_WIDTH, PAYMENT_HEIGHT);
+			jScrollPanePayment.setMaximumSize(size);
+			jScrollPanePayment.setMinimumSize(size);
+			jScrollPanePayment.setPreferredSize(size);
 		}
 		return jScrollPanePayment;
 	}
 
 	private JTable getJTablePayment() {
 		if (jTablePayment == null) {
-			jTablePayment = new JTable();
-			jTablePayment.setModel(new PaymentTableModel());
+			jTablePayment = new JTable(new PaymentTableModel());
 			jTablePayment.getColumnModel().getColumn(1).setMinWidth(PRICE_WIDTH);
 			jTablePayment.getColumnModel().getColumn(1).setMaxWidth(PRICE_WIDTH);
 		}
 		return jTablePayment;
 	}
 
+	private void setJTablePayment() {
+		jTablePayment.setModel(new PaymentTableModel());
+	}
+
 	private JScrollPane getJScrollPaneBalance() {
 		if (jScrollPaneBalance == null) {
 			jScrollPaneBalance = new JScrollPane();
 			jScrollPaneBalance.setViewportView(getJTableBalance());
 			jScrollPaneBalance.setVerticalScrollBarPolicy(ScrollPaneConstants.VERTICAL_SCROLLBAR_NEVER);
-			jScrollPaneBalance.setMaximumSize(new Dimension(PANEL_WIDTH, BALANCE_HEIGHT));
-			jScrollPaneBalance.setMinimumSize(new Dimension(PANEL_WIDTH, BALANCE_HEIGHT));
-			jScrollPaneBalance.setPreferredSize(new Dimension(PANEL_WIDTH, BALANCE_HEIGHT));
+			jScrollPaneBalance.setMaximumSize(PatientBillEdit.BALANCE_TABLE_SIZE);
```

Comment:
No need to fully qualify; make consistent with line 1012
```suggestion
			jScrollPaneBalance.setMaximumSize(BALANCE_TABLE_SIZE);
```